tpu/Dockerfile

```diff
@@ -6,8 +6,19 @@ FROM gcr.io/cloud-tpu-v2-images/libtpu:${LIBTPU_IMAGE_TAG} as libtpu
 FROM gcr.io/kaggle-images/python-tpu-tensorflow-whl:${TENSORFLOW_WHL_IMAGE_TAG} AS tensorflow_whl
 FROM gcr.io/kaggle-images/python:${BASE_IMAGE_TAG}
 
+ENV ISTPUVM=1
+
 COPY --from=libtpu /libtpu.so /lib
 
 COPY --from=tensorflow_whl /tmp/tensorflow_pkg/tensorflow*.whl /tmp/tensorflow_pkg/
 RUN pip install /tmp/tensorflow_pkg/tensorflow*.whl && \
-    rm -rf /tmp/tensorflow_pkg
\ No newline at end of file
+    rm -rf /tmp/tensorflow_pkg
+
+# https://cloud.google.com/tpu/docs/pytorch-xla-ug-tpu-vm#changing_pytorch_version
+RUN pip uninstall -y torch
```

Comment:
Make sure to clean the layer at the end of each `RUN` command otherwise the layer will include the compressed packages which makes the layer larger and increases the image size.

https://github.com/Kaggle/docker-python/blob/389cecd4683774c7ce33d39667dd788742e77816/Dockerfile.tmpl#L104-L107

I would recommend batching the RUN command and running a process to clean layer at the end.

Reply:
Done! Thanks.